src/gui/guiFormSpecMenu.cpp

```diff
@@ -2686,6 +2687,91 @@ void GUIFormSpecMenu::parseSetFocus(const std::string &element)
 		<< "'" << std::endl;
 }
 
+void GUIFormSpecMenu::parseMesh(parserData *data, const std::string &element)
+{
+	std::vector<std::string> parts = split(element, ';');
+
+	while (parts.size() < 7)
+		parts.push_back("");
```

Comment:
How about
```C++
parts.resize(std::max(parts.size(), 7);
```
?
Please move the code after the check below because the first four args are not optional.